roofit/roofitcore/inc/RooCmdConfig.h

```diff
@@ -72,7 +72,7 @@ class RooCmdConfig : public TObject {
 
   Bool_t ok(Bool_t verbose) const ;
 
-  const char* missingArgs() const ;
+  std::string missingArgs() const ;
```

Comment:
This is an interface change (returning a std::string instead of a char *). 
I agree is better since it avoids using a static TString. 
We should maybe write in the documentation and/or release notes

Reply:
Ok! I have added a commit where I mention this in the v6.28 release notes. But I guess there is no need to wait for the CI tests now, only touching the release notes file.